tests/e2e-agents/e2e_agents/ci_entrypoint.py

```
@@ -0,0 +1,207 @@
+
+def parse_labels() -> list[str]:
+    """Parse PR labels from environment."""
+    raw = os.environ.get("LABELS", "")
+    if not raw:
+        return []
+
+    # Try JSON array first (from GitHub Actions toJSON)
+    try:
+        labels = json.loads(raw)
+        if isinstance(labels, list):
+            return [str(l) for l in labels]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)